Makefile

```diff
@@ -154,7 +154,7 @@ endif
 ASFLAGS := -march=vr4300 -32 -no-pad-sections -Iinclude
 
 ifeq ($(COMPILER),gcc)
-  CFLAGS += -G 0 -nostdinc $(INC) -march=vr4300 -mfix4300 -mabi=32 -mno-abicalls -mdivide-breaks -fno-zero-initialized-in-bss -fno-toplevel-reorder -ffreestanding -fno-common -fno-merge-constants -mno-explicit-relocs -mno-split-addresses $(CHECK_WARNINGS) -funsigned-char
+  CFLAGS += -G 0 -nostdinc $(INC) -march=vr4300 -mfix4300 -mabi=32 -mno-abicalls -mdivide-breaks -fno-PIC -fno-common -ffreestanding -fbuiltin -fno-builtin-sinf -fno-builtin-cosf $(CHECK_WARNINGS) -funsigned-char
```

Comment:
asking for the sake of asking, why `-fno-PIC` rather than `-fno-pic`

I will look into `-fbuiltin -fno-builtin-sinf -fno-builtin-cosf` but if you feel like giving a short explanation I'd appreciate it

Reply:
- iirc PIC vs pic is just whether to use a big GOT or not, afaik disabling just one is sufficient but maybe disabling both is best..
- `-fbuiltin` followed by `-fno-builtin-X` enables all builtins except X, so here I enable builtins in general but disable the trig ones